README.md

```diff
@@ -272,6 +322,30 @@ on_success:
     chat_message_file: the-message-dir/msg.txt
 ```
 
+If using send to chat only and the `chat_message_file` parameter, the put step requires only one ["put inputs"]. For example:
+
+```yaml
+on_success:
+  put: gh-status
+  # the-message-dir: "output" of a previous task
+  inputs: [the-message-dir]
+  params:
+    state: success
```

Comment:
why do you need state if you just push to gchat?

Reply:
It's optional or you may want to send only for success state.

Reply:
I was confused. I though its under `Chat notification only` section.